pkg/rpcclient/actor/waiter.go

```diff
@@ -0,0 +1,201 @@
+package actor
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/block"
+	"github.com/nspcc-dev/neo-go/pkg/core/state"
+	"github.com/nspcc-dev/neo-go/pkg/neorpc"
+	"github.com/nspcc-dev/neo-go/pkg/neorpc/result"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/trigger"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+var (
+	// ErrTxNotAccepted is returned when transaction wasn't accepted to the chain
+	// even after ValidUntilBlock block persist.
+	ErrTxNotAccepted = errors.New("transaction was not accepted to chain")
+	ErrContextDone   = errors.New("waiter context done")
+)
+
+type (
+	// SimpleWaiter is an interface that enables transaction awaiting functionality
+	// for Actor instance.
+	SimpleWaiter interface {
```

Comment:
`RPCSimpleWaiter`? We have `RPCActor` for RPC things.

Reply:
`RPCPollingWaiter`?